assets/node-exporter/prometheus-rule.yaml

```diff
@@ -185,6 +186,7 @@ spec:
         and
           deriv(node_timex_offset_seconds{job="node-exporter"}[5m]) <= 0
         )
+        ) and on() absent(up{job="ptp-monitor-service"})
```

Comment:
@midu16 can you validate the job label value here? Looking at the telemetry data, I see that the `NodeOutOfPtpSync` alert fires with this value.